src/covid_model_seiir_pipeline/pipeline/forecasting/model/forecast_metrics.py

```diff
@@ -77,7 +76,7 @@ def compute_output_metrics(infection_data: pd.DataFrame,
 
 def compute_corrected_hospital_usage(all_age_deaths: pd.DataFrame,
                                      death_weights: pd.Series,
-                                     hospital_fatality_ratio: HospitalFatalityRatioData,
+                                     hospital_fatality_ratio,
```

Comment:
You deleted the type, but the argument is still valid?

Reply:
It's a series now.  Need to come back and update typing.